linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/policy/stig/shared.yml

```diff
@@ -15,28 +15,27 @@ checktext: |-
 
     Check the cron directories for scripts controlling the execution and notification of results of the file integrity application. For example, if AIDE is installed on the system, use the following commands:
 
-    $ ls -al /etc/cron.* | grep aide
+    $ sudo ls -al /etc/cron.* | grep aide
 
     -rwxr-xr-x 1 root root 29 Nov 22 2015 aide
 
-    $ grep aide /etc/crontab /var/spool/cron/root
+    $ sudo grep aide /etc/crontab /var/spool/cron/root
 
     /etc/crontab: 30 04 * * * root usr/sbin/aide
     /var/spool/cron/root: 30 04 * * * root usr/sbin/aide
 
     $ sudo more /etc/cron.daily/aide
 
     #!/bin/bash
-    {{{ aide_bin_path }}} --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil
-
-    If the file integrity application does not exist, or a script file controlling the execution of the file integrity application does not exist, or the file integrity application does not notify designated personnel of changes, this is a finding.
+    /usr/sbin/aide --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil
 
+    If the file integrity application does not exist, a script file controlling the execution of the file integrity application does not exist, or the file integrity application does not notify designated personnel of changes, this is a finding.
 fixtext: |-
     Configure the file integrity tool to run automatically on the system at least weekly and to notify designated personnel if baseline configurations are changed in an unauthorized manner. The AIDE tool can be configured to email designated personnel with the use of the cron system.
 
-    The following example output is generic. It will set cron to run AIDE daily and to send email at the completion of the analysis.
+    The following example output is generic. It will set cron to run AIDE daily and to send email at the completion of the analysis
 
     $ sudo more /etc/cron.daily/aide
 
     #!/bin/bash
-    {{{ aide_bin_path }}} --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil
```

Comment:
Should we revert to the jinja variable here instead?

Reply:
Due to the import processes we usually don't use Jinja in the PSC content.

Reply:
You can override the content on per product basis by using `rhel10.yml` for example.

Reply:
It's an investment for the future.

Reply:
Yeah, some manual work needs to happen for that to work. I think it should be fine for now. Let's merge it.